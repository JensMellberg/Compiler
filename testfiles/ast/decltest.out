Program
  List
    FuncDecl
      IdDecl ID="a"
      List
        IdParam
          IdDecl ID="b"
        IdParam
          IdDecl ID="c"
      Block
        List
