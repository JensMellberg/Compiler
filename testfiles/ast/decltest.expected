Program
  List
    FuncDecl
      IdDecl ID="a"
      List
        IdReadDecl
          IdDecl ID="b"
          Opt
        IdReadDecl
          IdDecl ID="c"
          Opt
      Block
        List
